filedescriptor/src/windows.rs

```diff
@@ -208,6 +213,43 @@ impl OwnedHandle {
             })
         }
     }
+
+    pub(crate) fn redirect_stdout_to_file(&self) -> anyhow::Result<Redirection> {
+        let std_original = unsafe{ GetStdHandle(STD_OUTPUT_HANDLE) } ;
```

Comment:
I think you want to `FileDescriptor::dup(std_original)` here so that you have a managed duplicate handle.  It may be slightly better to `FileDescriptor::new(std_original)` so that you take complete ownership of that handle; that is semantically closer to what this would do on unix.

Reply:
Have made this change to the code now